Expensify/App
# Show Loader when adding Workspace Edit

### Details
- Add loading flags for Avatar upload and form submission
- IMP: Added `isUploading` flag to AvatarWithImagePicker with uploadIndicator
- Replaced js loop to Animated.loop

### Fixed Issues

$ https://github.com/Expensify/App/issues/4487

### Tests
1. Tested by keeping the form contents the same
2. Updating the workspace name and image
3. Removing the workspace image
4. Submitting the form successfully
5. Uploading a large image



### QA Steps
1. Go to Workspace and click on the Avatar
2. Check button state when the values are not updated
3. Change the content/upload the image and see if values are updated
3. Test Removing the workspace image as well 
4. Submitting the form successfully
5. Uploading a large image



### Tested On


### Screenshots


#### Web
**v1**
https://user-images.githubusercontent.com/3069065/131385161-e70fc702-9ef7-4515-bacd-86ac728d5c03.mp4

**Updated**
https://user-images.githubusercontent.com/3069065/132609914-524caa8b-ead7-4ec1-97c4-6288d6575dd7.mp4

#### Mobile Web
**v1**
https://user-images.githubusercontent.com/3069065/131384341-a5d4da09-e5e7-4807-8cf9-3724c797d2c6.mp4

**Updated**
https://user-images.githubusercontent.com/3069065/132610024-500c89e5-83d0-4c05-91c6-61688fd223c1.mp4

#### Desktop
**v1**
https://user-images.githubusercontent.com/3069065/131384466-c488f178-844d-471b-b9b6-9ca68fa5de65.mp4

**Updated**
https://user-images.githubusercontent.com/3069065/132610083-e81b6468-c976-4ec2-83cc-f7932de947f3.mp4

#### iOS
**v1**
https://user-images.githubusercontent.com/3069065/131384510-c38701d8-74ae-49c2-a116-e50e2fdc19a5.mp4

**Updated**
https://user-images.githubusercontent.com/3069065/132610283-1a851e62-640f-496d-86d2-83376a6e7125.mp4

#### Android
**v1**
https://user-images.githubusercontent.com/3069065/131384679-c6b796b0-f49e-4e76-8d48-3991d2cac241.mp4

**Updated**
https://user-images.githubusercontent.com/3069065/132610438-29e4b86c-81aa-45fc-ad41-7d77e0ba6c92.mp4
